tests/network/rdp/test_rdp_for_exposed_vm_svc.py

```diff
@@ -75,6 +75,7 @@ def rdp_pod(workers_utility_pods, rdp_vm):
     ],
     indirect=True,
 )
+@pytest.mark.skip(reason="CNV-43997")
```

Comment:
please use `xfail(...., run=False)` and not skip
also, please provide a reason (not just the jira id); quarantined tests will not be executed but should not be skipped